nonCisMaleInCs/DetailsView.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {StyleSheet, View, Text, Image, Button, Alert, TouchableOpacity} from 'react-native';
+// import { Icon } from 'react-native-elements' 
+// import { Header } from 'react-native-elements';
+// import Icon from 'react-native-vector-icons/FontAwesome';
+import editIcon from './edit.png';
```

Comment:
let's put images in a folder called `/assets/images`

Reply:
Done

Reply:
Hi Temi! Please also remove the `edit.png` and `nophoto.png` that exists in the `nonMaleInCs` folder, since that's of no use.